src/css/main.less

```diff
@@ -26,6 +26,7 @@
 @import "components/filter-box.less";
 @import "components/loading-bar.less";
 @import "components/health-bar.less";
+
```

Comment:
Oh! You're right! I didn't thought about that. How convenient. The modern world is just too complex for me.. ;)